packages/expo-sensors/src/PlatformSensorModule.ts

```diff
@@ -0,0 +1,27 @@
+import { EventEmitter } from 'expo-core';
+
+export default class PlatformSensorModule {
+  emitter = new EventEmitter({} as any);
```

Comment:
What listens to this emitter? Looking through this PR, it looks like we want to indirectly wire up the EventEmitter in the module "frontend" with DOM events. Sometimes adding an extra emitter in the "backend" might help but in this PR it seems like we could override `addListener` in the PlatformSensorModule subclasses to add DOM event handlers, instead of overriding start/stopObserving.

Also, if we keep it can you rename this to `__emitter` so that it's "protected" (rather than public, but still accessible if we need it in tests)